src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/ManagedDatabaseCrudScenarioTests.ps1

```diff
@@ -0,0 +1,239 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+	.SYNOPSIS
+	Tests creating a managed database
+#>
+function Test-CreateManagedDatabase
+{
+	# Setup
+	$rg = Create-ResourceGroupForTest
```

Comment:
My recommendation was to use a constant for resource group and subnet name so that the same subnet can be used across multiple tests, so that only the first MI creation is slow. Note that this means CreateAndGetVirtualNetworkForManagedInstance should be able to immediately return the subnet if it already exists.